src/components/internals/rich-text-body/multi-dropdown.js

```diff
@@ -63,15 +60,7 @@ const MultiDownshift = props => {
 
   return (
     <Downshift onSelect={props.onSelect} itemToString={itemToString}>
-      {({
-        getToggleButtonProps,
-        getMenuProps,
-        isOpen,
-        getItemProps,
-        toggleMenu,
-      }) => {
-        const toggleButtonProps = omit(getToggleButtonProps(), propsToRemove);
```

Comment:
no need to omit because now we can just use `onClick`, as we are preventing the onBlur at the higher level.